Modules/FindNetCDF.cmake

```diff
@@ -30,120 +31,42 @@ if (NETCDF_INCLUDES AND NETCDF_LIBRARIES)
   set (NETCDF_FIND_QUIETLY TRUE)
 endif (NETCDF_INCLUDES AND NETCDF_LIBRARIES)
 
-if(NETCDF)
-  set(NETCDF_DIR ${NETCDF})
-else(NETCDF)
-  # This is tricky, WRF users often have "export NETCDF4=1" in their environment (from old days)
-  if(DEFINED ENV{NETCDF4})
-    set(NETCDF_DIR $ENV{NETCDF4})
-  elseif(DEFINED ENV{NETCDF_DIR})
-    set(NETCDF_DIR $ENV{NETCDF_DIR})
-  elseif(DEFINED ENV{NETCDF_HOME})
-    set(NETCDF_DIR $ENV{NETCDF_HOME})
-  elseif( DEFINED ENV{NETCDF} )
-    set(NETCDF_DIR $ENV{NETCDF})
-  elseif(DEFINED ENV{SSEC_NETCDF4_DIR})
-    set(NETCDF_DIR $ENV{SSEC_NETCDF4_DIR})
-  elseif(DEFINED ENV{SSEC_NETCDF_DIR})
-    set(NETCDF_DIR $ENV{SSEC_NETCDF_DIR})
-  endif()
-endif(NETCDF)
-
-if(NOT NETCDF_FORTRAN)
-  if(DEFINED ENV{NETCDF_FORTRAN})
-    set(NETCDF_FORTRAN $ENV{NETCDF_FORTRAN})
-  elseif(DEFINED ENV{NETCDF_FORTRAN_DIR})
-    set(NETCDF_FORTRAN $ENV{NETCDF_FORTRAN_DIR})
-  endif(DEFINED ENV{NETCDF_FORTRAN})
-endif(NOT NETCDF_FORTRAN)
-
 find_path (NETCDF_INCLUDES netcdf.h
-  HINTS ${NETCDF_DIR}/include $ENV{SSEC_NETCDF_DIR}/include )
-
-find_program (NETCDF_META netcdf_meta.h
-    HINTS ${NETCDF_INCLUDES} ${CMAKE_INSTALL_PREFIX}
-    )
-if (NETCDF_META)
-  file (STRINGS ${NETCDF_META} NETCDF_VERSION REGEX "define NC_VERSION_MAJOR")
-  string (REGEX REPLACE "#define NC_VERSION_MAJOR " "" NETCDF_VERSION ${NETCDF_VERSION})
-  string (REGEX REPLACE "\\/\\*\\!< netcdf-c major version. \\*\\/" "" NETCDF_VERSION ${NETCDF_VERSION})
-  string (REGEX REPLACE " " "" NETCDF_VERSION ${NETCDF_VERSION} )
-  if(${NETCDF_VERSION} GREATER "3")
-    set(NETCDF_F90 "YES")
-  endif()
-endif (NETCDF_META)
-
-find_library (NETCDF_flib
-     names ${CMAKE_STATIC_LIBRARY_PREFIX}netcdff${CMAKE_STATIC_LIBRARY_SUFFIX}
-        ${CMAKE_SHARED_LIBRARY_PREFIX}netcdff${CMAKE_SHARED_LIBRARY_SUFFIX}
-     HINTS
-        ${NETCDF_DIR}/lib
-        ${NETCDF_FORTRAN_DIR}/lib
-        ${NETCDF_FORTRAN}/lib
-        ${NETCDF_FORTRAN_ROOT}/lib
-)
+  HINTS ${NETCDF_DIR}/include )
```

Comment:
The hint uses NETCDF_DIR. I assume the mother-CMakeLists.txt checks for the environment variable NETCDF and then sets the internal cmake variable NETCDF_DIR?